Patch pyproject for tarball and not dynamic version
Fix by rick <pentoo@gyiwr.tf>

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -2,7 +2,6 @@
 build-backend = "setuptools.build_meta"
 requires = [
   "setuptools>=61.2",
-  "setuptools_scm",
 ]
 
 [project]
@@ -32,9 +31,7 @@ classifiers = [
   "Programming Language :: Python :: 3.12",
   "Topic :: Internet :: WWW/HTTP",
 ]
-dynamic = [
-  "version",
-]
+version = "2.7"
 dependencies = [
   "Django>=3.2",
 ]

